plugin/completion.py

```diff
@@ -101,6 +191,17 @@ def resolve(completion_list: sublime.CompletionList, items: List[sublime.Complet
     sublime.set_timeout(lambda: completion_list.set_completions(items, flags))
 
 
+def deep_escape(what):
+    if isinstance(what, dict):
+        return {key: deep_escape(value) for key, value in what.items()}
+    elif isinstance(what, list):
+        return [deep_escape(value) for value in what]
+    elif isinstance(what, str):
+        return html.escape(what)
+    else:
+        return what
```

Comment:
I don't think this function is needed (see other comment).

Reply:
deep_escepe is removed